src/core/caniuse.js

```diff
@@ -133,18 +141,43 @@ function validateBrowsers({ caniuse }) {
 }
 
 async function processJson(json, { feature }) {
+  /** @type {Array} */
   const results = json.result;
-  const out = results.map(({ browser: browserId, version, caniuse }) => {
-    const { name } = BROWSERS.get(browserId);
+  const groups = new Map([
+    ["desktop", []],
+    ["mobile", []],
+  ]);
+  const toBrowserCell = browserCellRenderer(feature);
+  results.reduce(toBrowserCell, groups);
+  const out = [...groups.keys()]
```

Comment:
Can use `.entries()` instead (or shorthand `...groups.filter().map()`)

Reply:
> Can use .entries() instead 

Also returns an iterator :( 

> or shorthand ...groups.filter().map())

No sure what you mean? move it to another function?

Reply:
`out = [...groups].filter(([key, values]) => {})`